packages/react-storage/src/components/StorageBrowser/views/LocationActionView/hooks/utils.ts

```diff
@@ -0,0 +1,60 @@
+import { IconVariant } from '../../../context/elements';
+import { TaskStatus } from '../../../tasks';
+import { getTaskCounts } from '../../../controls/getTaskCounts';
+import { TaskCounts } from '../../../controls/types';
+
+export interface GetActionViewTaskStatuses {
```

Comment:
The naming here is confusing, think we can just declare the return of  `getActionViewTaskStatuses` inline  since we aren't using this interface anywhere else?

Reply:
updated and simplified